app/views/taxCodeChange/TaxCodeComparisonView.scala.html

```diff
@@ -74,7 +78,7 @@
 
 @taxCodeRecordPod(taxCodeRecord: Option[TaxCodeRecord], identifier: String) = {
     @{taxCodeRecord match {
-        case Some(taxCodeRecord) => formattedTaxCodeForComparison(taxCodeRecord, identifier, taxCodeChangeViewModel.scottishTaxRateBands, appConfig)
+        case Some(taxCodeRecord) => FormattedTaxCodeForComparison(taxCodeRecord, identifier, taxCodeChangeViewModel.scottishTaxRateBands, appConfig)
```

Comment:
@Sachin300793 just checking, should this have a capital F?

Reply:
Changed the file name from formattedTaxCodeForComparison to FormattedTaxCodeForComparison